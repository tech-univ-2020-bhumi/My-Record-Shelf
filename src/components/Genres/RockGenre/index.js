import React from 'react';
import getRecord from '../../getRecords';
import useRecords from '../../../hooks/useRecords';

function RockGenre(props) {
  const { testId } = props;
  const [records, getRecords] = useRecords();
  const rockArr = getRecord('rock', records);
  console.log(rockArr);
  return (
    <div>
      <div>rock</div>
      <div />
    </div>
  );
}

export default RockGenre;
