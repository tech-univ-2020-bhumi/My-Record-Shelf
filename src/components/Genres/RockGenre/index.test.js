/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import RockGenre from './index';

describe('The RockGenre component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<RockGenre testId="123" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
