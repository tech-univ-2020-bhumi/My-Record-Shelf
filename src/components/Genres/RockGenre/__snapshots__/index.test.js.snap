// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The RockGenre component should render correctly 1`] = `
<DocumentFragment>
  <div>
    <div>
      rock
    </div>
    <div />
  </div>
</DocumentFragment>
`;
